scripts/mainmenu.js

```diff
@@ -387,8 +387,8 @@ class MainMenuController {
 	 */
 	static onSafeguardMapChange(mapName) {
 		UiToolkitAPI.ShowGenericPopupOkCancel(
-			$.Localize('#MOM_MB_Safeguard_Map_Change_Title'),
-			`You are trying to change map to ${mapName} while your timer is still running, do you want to continue?\n\n<span class='text-sm text-italic'>(You can turn this off in Settings → Gameplay → Safeguards)</span>`,
+			$.Localize('#Safeguard_MapQuitToMenu'),
+			$.Localize('#Safeguard_MapQuitToMenu_Message'),
```

Comment:
Is the map name no longer used here?

Reply:
Think I was just being a bit lazy here, added it back.